### PR TITLE
Create a Docker container for testing glome-login

### PR DESCRIPTION
**Pull request by vvidic**
Build and start the container using:

    $ docker build -t glome -f kokoro/docker/Dockerfile .
    $ docker run -d -p 2023:2023 glome

Connect to the container using netcat or socat, for example:

    $ socat tcp-connect:localhost:2023 file:`tty`,raw,echo=0

Get the authorization code by running:

    $ docker exec b4b6e14f1d2c /usr/local/bin/glome login --key /usr/local/etc/glome/private.key http://...

### CODE REVIEW

File: kokoro/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN kokoro/rodete/fetch_dependencies.sh
+RUN rm -rf build
+RUN meson build
+RUN meson compile -C build
+RUN meson test --print-errorlogs -C build
+RUN meson install -C build
+
+FROM docker.io/library/debian:bullseye
+COPY --from=build /usr/local /usr/local
+COPY --from=build /app/kokoro/docker/glome-start /usr/local/sbin
```
Comment (burgerdev) [inline kokoro/docker/Dockerfile:13]: Any reason why this is not just `COPY kokoro/docker/glome-start /usr/local/sbin`?

File: kokoro/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+RUN apt-get update
+RUN apt-get -y install socat xxd libglib2.0-0
```
Comment (burgerdev) [inline kokoro/docker/Dockerfile:15]: These should usually be combined into one `RUN` block. 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

**Review by burgerdev — COMMENTED** [review]
Neat! Could you please also commit the documentation from the PR description (or something similar)?

File: kokoro/docker/Dockerfile (hunk 2 above)
Comment (vvidic) [reply kokoro/docker/Dockerfile:15]: Done, thanks!

File: cli/commands.c [hunk 3]
```diff
@@ -371,6 +371,9 @@ int login(int argc, char **argv) {
   }
 
   char *path = argv[optind];
+  if (strstr(path, "http") == path) {
+    path = strstr(path, "/v1/");
+  }
```
[the remark this answers is not in the document]
Comment (vvidic) [reply cli/commands.c:376]: Makes sense, yes.

File: kokoro/docker/Dockerfile (hunk 1 above)
Comment (vvidic) [reply kokoro/docker/Dockerfile:13]: Done.

**Comment by vvidic** [comment]
Ok, added the commands to the README along with the support for SSH/PAM :)

**Review by burgerdev — APPROVED** [review]

**Pull request merged**